ruby-openai.gemspec

```
@@ -25,7 +25,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "httparty", ">= 0.18.1"
+  spec.add_dependency "faraday", ">= 2"
```

@alexrudall iIs there an actual requirement on faraday `2`, or could this dependency be relaxed to allow older versions of faraday? I'm asking because I wanted to test this branch out in an existing codebase that uses an older version of faraday, and since it wasn't easy to upgrade everything else to faraday 2 I just tested with `1.10` and everything seemed worked fine (at least for my use case).
```suggestion
  spec.add_dependency "faraday", "< 3"
```

---

No requirement that I can see, changed to >= 1